cumulusci/utils/xml/robot_xml.py

```diff
@@ -30,17 +31,15 @@ def _format_metric(metric_name: str, value: float):
 
 def _perf_formatter(perfsummary: PerfSummary) -> str:
     """Default formatter for perf summaries"""
-    metrics = perfsummary.metrics
-    elapsed_time = metrics.get("total_time")
-    assert elapsed_time  # TODO: Remove this
-    result = f"`{perfsummary.name}` - "
 
     other_metrics = [
         _format_metric(metric, value)
         for metric, value in perfsummary.metrics.items()
         if metric != "total_time"
     ]
 
+    result = f"`{perfsummary.name}` - "
```

Comment:
Why are you adding backticks around the name? They don't change how anything is formatted and add visual noise IMO. 
```
2021-01-05 17:27:30: `Example performance test` - Elapsed Time: 3.0s , number of records:: 100.0 
```

Reply:
Removed.